static/src/javascripts/bootstraps/common.js

```diff
@@ -133,7 +133,8 @@ define([
             },
 
             initialiseStickyHeader: function () {
-                if (config.switches.viewability && (config.page.isDev || (!config.page.isDev && config.page.contentType !== 'Interactive'))) {
+                var isCode = (config.page.host) ? config.page.host.match('code') : false;
+                if (config.switches.viewability && (isCode || config.page.contentType !== 'Interactive')) {
```

Comment:
Would it be better to set an `isPreProd` flag and make it available to clientside?  Then wouldn't need logic for `isCode` here and wouldn't need to distinguish between Code and Dev.